operator/config/samples/karmada.yaml

```diff
@@ -31,19 +31,15 @@ spec:
       serviceSubnet: 10.96.0.0/12
     karmadaAggregatedAPIServer:
       imageRepository: docker.io/karmada/karmada-aggregated-apiserver
-      imageTag: v1.7.0
```

Comment:
We might not need to remove the version here.
I think even though `karmada-operator` can get the version, it is still recommended to declare the image version.

Reply:
done